src/test/java/millom/sandbox/mapper/JdkMapperTest.java

```diff
@@ -0,0 +1,50 @@
+package millom.sandbox.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.IOException;
+import millom.sandbox.CustomException.JdkException;
+import millom.sandbox.Utility.JsonUtility;
+import millom.sandbox.pojos.ReleaseNames;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class JdkMapperTest {
+
+  private final static JdkMapper jdkMapper = new JdkMapper();
+
+  @Test
+  @DisplayName("TEST JSON STRING MAPPING CORRECT TO THE RELEASE NAMES OBJECT")
+  void deserializeReleaseNames() throws JdkException, IOException {
+    ReleaseNames response = jdkMapper.deserializeReleaseNames(
+        JsonUtility.getSampleResponse("JdkJsonResources/releaseNamesArmPage0.json"));
+    assertThat(response).isNotNull();
+    assertThat(response.releases()).hasSize(10);
+    assertThat(response.releases().get(2)).isEqualTo("jdk-18+36");
```

Comment:
Should be `get(3)` I think, as `get(2)`returns "jdk-18.0.1+10"? Or change the string to "jdk-18.0.1+10" maybe?

Reply:
This is happened because the "JdkJsonResources/releaseNamesArmPage0.json" file content is changed.
Now changed the string to "jdk-18.0.1+10". And the test is passing now.